build.sbt

```diff
@@ -55,6 +60,8 @@ val timeBenchmark =
     .enablePlugins(JmhPlugin)
     .settings(commonSettings: _*)
     .settings(
+      // FIXME: does not compile with dotty
+      crossScalaVersions := List(scalaVersion.value),
```

Comment:
I added the `-noPredef` option to avoid ambiguities / name clashes with standard collections but I don’t think we need it. Alternatively, we could fix Dotty’s Java parser by generating `scala.Predef.???` instead of just `???`.